installer/install.py

```diff
@@ -222,19 +224,15 @@ def validate_sd():
     if not found_sdcard:
         report_failure(f"SD card was not mounted in the expected location. If the SD card is inserted, please file a bug.")
 
-def debug_wifi():
-    os.system("/userdata/x1plus/debug_wifi.sh")
-
 validate_sd()
 
 df = shutil.disk_usage("/mnt/sdcard")
 if df.free < demand_free_space:
     report_failure(f"The SD card does not have enough free space. Make sure that at least {demand_free_space//(1024*1024)} MB are available.")
 
 if not os.path.isfile("/sys/devices/virtual/misc/loop-control/dev"):
-    if os.system(f"insmod {installer_path}/loop.ko") != 0:
+   if os.system(f"insmod {installer_path}/loop.ko") != 0:
```

Comment:
local Python style is 4 spaces, not 3 space indentation (and whitespace is significant in Python!)